data/pc/common/features.json

```diff
@@ -359,5 +359,35 @@
         "version": "1.8_major"
       }
     ]
+  },
+  {
+    "name": "itemSerializationWillOnlyUsePresent",
+    "description": "item serialization in [even] newer versions uses present field [exclusively] to show nullability rather than sending blockId as -1",
+    "versions": ["1.14_major", "latest"]
+  },
+  {
+    "name": "itemSerializationAllowsPresent",
+    "description": "item serialization in newer versions uses present field to show nullability rather or sending blockId as -1 can be used",
+    "versions": ["1.13_major", "latest"]
+  },
+  {
+    "name": "itemSerializationUsesBlockId",
+    "description": "item serialization in older versions uses blockId field to show nullability by setting blockId to -1",
+    "versions": ["1.8_major", "1.12_major"]
+  },
+  {
+    "name": "preFlattening",
+    "description": "in older versions, enchant ids were used to store enchants in nbt",
+    "versions": ["1.8_major", "1.12_major"]
+  },
+  {
+    "name": "theFlattening",
```

Comment:
pick a feature name related to what this does (like enchant)

Reply:
the flattening effected a lot of things, too many to point to one: https://minecraft.fandom.com/wiki/Java_Edition_1.13/Flattening

Reply:
yes, I still think we should have one feature for one thing
features shouldn't be used only to identify a version range

btw I don't think this specific feature is that related with theFlattening (unlike blockStateId feature for example)